server/channels/db/migrations/postgres/000126_add_mfa_used_ts_to_users.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE Users ADD COLUMN IF NOT EXISTS MfaUsedTimestamps VARCHAR(35) NOT NULL DEFAULT '';
```

Comment:
It seems like we are actually storing an array of integers, and then converting them into a comma separated string and storing a text. Why not have `integer[]` as a data type?

Reply:
That would be `integer[]` for postgres and `JSON` for MySQL?

Reply:
Indeed, in that case I'd suggest to go with jsonb for postgres, json for MySql. We already have an example for that: see the participants column in Threads table. `(s *SqlThreadStore) MaintainMembership` method contains code to add a participant.

The thing with database schema changes are that once it's done, it's nearly impossible to change them later. So it's a good practice to spend some time to get it right.